infor-design/enterprise
# 6868 - New Icons and New Icon Sizes

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

UPDATE: This PR is ready to review and test. I am updating the images so i will fix the checks. Ignore the check about the package.json's that is added as this is just extra protection requested by @danielortiz1982 

The figma file has been updated to include a lot of new icons and a change that adds 2px padding on the top and the bottom of icons. That has the effect of making all icons smaller.

In the code anyplace that we have a width/height on icons that is not 18px it could look wierd. If smaller than 18px we should change it to 18px, if larger than 18px we may need to change the size up/down to what looks good depending on the component.

I made changed for everything i could find here on this PR, but feel free to push any fixes as well.

**Related github/jira issue (required)**:
Fixes #6868 
https://jira.infor.com/browse/IDS-509 

**Steps necessary to review your pull request (required)**:
- pull this branch
- take zip https://infor-design-assets-downloads.s3.amazonaws.com/archives/dist.zip (may need new one ask Artem)
- copy zip contents to node_modules/ids-identity/dist
- `npm run start`

**Some Ways to Test**
- go to http://localhost:4000/components/icons/example-index.html
- notice the new icon sections and counts
- right click any icon and change the css color in the dev tools to something like "red" and make sure every icon takes the color fill
- look at http://localhost:4000/ at all things icon related and compare to https://main-enterprise.demo.design.infor.com/ things will be smaller but should look aligned ect..We can make some updates to the icons (design) or some css fixes as needed
- test http://localhost:4000/components/rating/example-index.html
- make sure more icon looks bigger
- test http://localhost:4000/components/editor/example-index.html
- http://localhost:4000/components/alerts/example-index.html (all icons should show up)
- test any other places you think of

Last Test Round Ensure these are solved
- Not Getting Fill Color: icon-violence, icon-week-filled
- Still "missing": icon-favicon-black (has a black clip color)
- Icon More: Still Hard to See (can it be more bigger)
- Looks Dimmer: #icon-right-text-align (see editor)
- X icon seems alot bigger than the search (see search)
- some icons missing on the alerts for example: #icon-success-alert and #icon-alert-alert(see alert)

**Included in this Pull Request**:


**Before Merging**:
 Check with @tmcconechy
